sdk/eventhubs/azure-messaging-eventhubs/pom.xml

```diff
@@ -42,7 +42,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core-amqp</artifactId>
-      <version>2.5.0</version> <!-- {x-version-update;com.azure:azure-core-amqp;dependency} -->
+      <version>2.6.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure:azure-core-amqp;dependency} -->
```

Comment:
I don't see changes to Service Bus. Are we not supporting identifiers for Service Bus clients?

Reply:
Not now. Jesse is preparing this.